### PR TITLE
feat: create pre-configured role with m-api access when seeding db

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Create a pre-configured role with Management API access when seeding the database for a better Logto Management API integration experience

### Note
Since this feature is only for new user, so we don't need an alteration script.

## Testing

After the Logto instance setup:

image

image

## Checklist

- [x] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [x] necessary TSDoc comments

### CODE REVIEW

File: .changeset/angry-tables-repeat.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@logto/schemas": minor
+"@logto/cli": minor
+---
+
+create a pre-configured Logto Management API access role on database seeded for a better Logto Management API integration experience
```
[the remark this answers is not in the document]
Comment (gao-sun) [reply .changeset/angry-tables-repeat.md:6]: I think the reason "for a better experience" can be omitted here - mention it in the pull request description is already enough

**Review by gao-sun — COMMENTED** [review]
The "database seeded" expression is weird, can you update it?

File: packages/cli/src/commands/database/seed/tables.ts [hunk 2]
```diff
@@ -150,6 +151,12 @@ export const seedTables = async (
   await seedOidcConfigs(connection, defaultTenantId);
   await seedAdminData(connection, defaultManagementApi);
 
+  /**
+   * Create a pre-configured role for the Logto Management API access
+   * in the default tenant (the default tenant is the only tenant for the OSS version, and the initial tenant for cloud).
+   */
+  await seedPreConfiguredManagementApiAccessRole(connection, defaultTenantId);
```
Comment (simeng-li) [inline packages/cli/src/commands/database/seed/tables.ts:158]: How about putting the logic in the seedAdminData(connection, defaultManagementApi)

Comment (gao-sun) [reply packages/cli/src/commands/database/seed/tables.ts:158]: seedAdminData will also be used for the admin tenant, which is not expected

**Review by gao-sun — APPROVED** [review]

**Pull request merged**